conode/Makefile

```diff
@@ -52,10 +57,14 @@ docker_clean:
 
 # The suffix on conode exe is the result from: echo `uname -s`.`uname -m`
 # so that we can find the right one in the wrapper script.
-bindist:
+# This is in it's own rule because the Docker build needs it also.
+exe/conode.Linux.x86_64:
+	GOOS=linux GOARCH=amd64 go build $(flags) -o $@
```

Comment:
But now you miss the `OUTPUT_DIR`, now? Wouldn't it be simpler to have `docker_dev` depend on `bindist` only? Even if you create 4 binaries too many?

Reply:
OK, looking through the code I see the light: you use `exe/` as a directory for the conode-binary, and `conode-...` as directory for the bindist.

I think I would still keep the x86-binary in bindist and then pass `OUTPUT_DIR` to Dockerfile-dev. But if you prefer it this way, it's also possible.

Reply:
I prefer it this way. Docker doesn't need any args, it knows where to find the thing Make just made for it.